<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0.149254 rgba(6, 0, 172, 69), stop:0.375 rgba(64, 7, 150, 69), stop:0.393035 rgba(28, 0, 255, 145), stop:0.45 rgba(0, 75, 159, 227), stop:0.477581 rgba(111, 71, 255, 130), stop:0.487562 rgba(56, 23, 156, 130), stop:0.512438 rgba(103, 14, 236, 181), stop:0.572139 rgba(21, 25, 121, 255), stop:0.60199 rgba(31, 37, 178, 255), stop:0.625 rgba(36, 25, 184, 69), stop:1 rgba(34, 0, 158, 69))</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="mainLayout" columnstretch="0,0">
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton5050">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>50/50</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonCallAFriend">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>Call a friend</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButtonAskTheAudience">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>Ask the audience</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="labelSum200000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: yellow;</string>
          </property>
          <property name="text">
           <string>15. 200000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum100000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>14. 100000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum50000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>13. 50000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum20000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>12. 20000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum10000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>11. 10000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum5000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: yellow;</string>
          </property>
          <property name="text">
           <string>10. 5000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum2500">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>9. 2500 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum2000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>8. 2000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum1500">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>7. 1500 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum1000">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>6. 1000 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum500">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: yellow;</string>
          </property>
          <property name="text">
           <string>5. 500 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum400">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>4. 400 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum300">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>3. 300 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum200">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>2. 200 euro</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelSum100">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string>1. 100 euro</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QPushButton" name="pushButtonStartGame">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Start Game</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>495</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QGraphicsView" name="audienceBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>176</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border-color: rgb(255, 255, 255);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="callerBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>176</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
color: white;
border-color: rgb(0, 0, 0);
</string>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>495</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonNextQuestion">
        <property name="styleSheet">
         <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
        </property>
        <property name="text">
         <string>Next question</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelQuestionText">
          <property name="styleSheet">
           <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonD">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonC">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonA">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;; color: white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonB">
            <property name="styleSheet">
             <string notr="true">font: 10pt &quot;MS Shell Dlg 2&quot;;
color: white;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
